backend/src/test/resources/application-default.yaml

```diff
@@ -18,6 +18,9 @@ logging:
     # org.hibernate.type: TRACE
     # Always have our own debug logging turned on in tests:
     gov.cdc.usds: DEBUG
+simple-report:
+  authorization:
+    role-prefix: "SR-UNITTES-TENANT:"
```

Comment:
UNITTEST ?

Reply:
😞 
Not that it actually makes a functional difference but yes, noticed that one after pushing last night.